errors.go

```diff
@@ -280,3 +280,40 @@ func Cause(err error) error {
 	}
 	return err
 }
+
+// Stack returns the earliest/deepest StackTrace attached to any of
+// the errors in the chain of Causes. If the error does not implement
+// Cause, the original error will be returned. If the error is nil,
+// nil will be returned without further investigation.
+
```

Comment:
The docstrings should not have a space between them an the function. I don’t know for sure this would break godoc, but convention suggest we strictly enforce it.

Reply:
OK. I'd remove it. Thank you.